dashboard/test/ui/runner.rb

```diff
@@ -313,7 +320,9 @@ def output_synopsis(output_text)
 
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    if $options.log_dev
+      HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    end
```

Comment:
What is the scenario in which someone *should* be using this flag? I would think in most cases if I'm rerunning a feature that failed as part of DTT from the test machine, logging to the same places the DTT logged seems desirable.